packages/assets/build/scss/global.scss

```diff
@@ -4,6 +4,10 @@
 
 @forward "layout";
 
+@forward "paths" with (
+  $assets-path: "https://unpkg.com/@massds/mayflower-assets@10.1.0/static"
```

Comment:
can the version be updated by the released version?

Reply:
Can we also link to the seal image CDN https://unpkg.com/@massds/mayflower-assets@[version]/static/images/logo/stateseal.png instead of using base64 embed in react headers?

Reply:
I've committed a fix to always point the version to the latest published.